src/components/hamburger/Sidebar.tsx

```diff
@@ -26,60 +28,36 @@ export const Sidebar = ({ isOpen }: SidebarProps) => {
     <AnimatePresence>
       {isOpen && (
         <motion.aside
-          className="fixed top-0 right-0 z-40 w-64 h-full shadow-sidebar bg-card-grey rounded-30"
-          variants={variants}
+          className="fixed top-0 right-0 z-40 w-72 h-full bg-card-grey shadow-card"
+          variants={{
+            open: { opacity: 1, x: 0 },
+            closed: { opacity: 0, x: '100%' },
+          }}
           initial="closed"
           animate="open"
           exit="closed"
           transition={{ type: 'spring', bounce: 0.25 }}
         >
-          <nav className="flex flex-col items-end justify-center h-full gap-5 mr-10 font-semibold uppercase text-end font-numbers ">
-            <a
-              href="#o-plese"
-              onClick={handleLinkClick}
-              className="mb-4 underline"
-            >
-              O plese
-            </a>
-            <a href="#program" onClick={handleLinkClick} className="mb-4">
-              Program
-            </a>
-            <a href="#vstupenky" onClick={handleLinkClick} className="mb-4">
-              Vstupenky
-            </a>
-            <a
-              href="#doprovodny-program"
-              onClick={handleLinkClick}
-              className="mb-4 "
-            >
-              Doprovodný program
-            </a>
-            <a href="#gallery" onClick={handleLinkClick} className="mb-4">
-              Galerie
-            </a>
-            <a href="#kontakty" onClick={handleLinkClick} className="mb-4">
-              Kontakt
-            </a>
-            <a href="#partneri" onClick={handleLinkClick} className="mb-4">
-              Partneři
-            </a>
-            <div className="flex gap-7">
-              <a
-                href="https://www.instagram.com/studentskaunieutb/"
-                target="_blank"
-                rel="noreferrer"
-                className="mb-4"
-              >
-                <img src={igIcon} alt="Instagram" />
-              </a>
+          <nav className="flex flex-col items-end justify-center h-full p-10">
+            {NavItems.map((item) => (
               <a
-                href="https://www.facebook.com/events/752918923323142?ref=newsfeed"
-                target="_blank"
-                rel="noreferrer"
-                className="mb-4"
+                key={item.id}
+                href={`#${item.id}`}
+                className={`block p-2 mb-4 ${activeNavItem === item.id ? 'underline' : ''} text-xl font-numbers font-semibold text-right`}
+                onClick={(e) => handleLinkClick(e, item.id)}
```

Comment:
remove the handleLinkClick function as I mentioned above and set the active item directly here